From 5f94e07588e06f03c5b12d3eff069ea5423bedb0 Mon Sep 17 00:00:00 2001
From: Will Thompson <will@willthompson.co.uk>
Date: Thu, 30 May 2024 11:11:03 +0100
Subject: [PATCH] GTK: Add 4.0.6 release to appstream metainfo

---
 gtk/transmission-gtk.metainfo.xml.in | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/gtk/transmission-gtk.metainfo.xml.in b/gtk/transmission-gtk.metainfo.xml.in
index 599fe08ac..0ce422978 100644
--- a/gtk/transmission-gtk.metainfo.xml.in
+++ b/gtk/transmission-gtk.metainfo.xml.in
@@ -52,6 +52,9 @@ Copyright 2017 Endless Mobile, Inc.
     <content_attribute id="social-info">mild</content_attribute>
   </content_rating>
   <releases>
+    <release date="2025-01-27" version="4.1.0">
+      <url>https://github.com/transmission/transmission/releases/tag/4.1.0</url>
+    </release>
     <release date="2023-12-06" version="4.0.5" />
     <release date="2023-08-23" version="4.0.4" />
     <release date="2023-04-14" version="4.0.3" />
-- 
2.39.2

