From e1d4dbb956010138fb1d12c9d52d886abe651c89 Mon Sep 17 00:00:00 2001
From: bbhtt <bbhtt.zn0i8@slmail.me>
Date: Fri, 3 Mar 2023 17:42:37 +0530
Subject: [PATCH] Add release dates

---
 gtk/transmission-gtk.metainfo.xml.in | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/gtk/transmission-gtk.metainfo.xml.in b/gtk/transmission-gtk.metainfo.xml.in
index e39180776..4310b7945 100644
--- a/gtk/transmission-gtk.metainfo.xml.in
+++ b/gtk/transmission-gtk.metainfo.xml.in
@@ -38,4 +38,9 @@ Copyright 2017 Endless Mobile, Inc.
   <content_rating type="oars-1.1">
     <content_attribute id="social-info">mild</content_attribute>
   </content_rating>
+  <releases>
+    <release date="2023-03-16" version="4.0.2" />
+    <release date="2023-02-23" version="4.0.1" />
+    <release date="2023-02-08" version="4.0.0" />
+    <release date="2020-05-03" version="3.00" />
+  </releases>
 </component>
--
2.39.2

