diff --git a/gtk/transmission-gtk.appdata.xml.in b/gtk/transmission-gtk.appdata.xml.in
index 169bdcd..3bc367c 100644
--- a/gtk/transmission-gtk.appdata.xml.in
+++ b/gtk/transmission-gtk.appdata.xml.in
@@ -35,4 +35,7 @@ Copyright 2017 Endless Mobile, Inc.
     </screenshot>
   </screenshots>
   <update_contact>info_AT_transmissionbt.com</update_contact>
+  <releases>
+    <release date="2020-05-03" version="3.00" />
+  </releases>
 </component>
